### PR TITLE
Update pre-commit configuration

### PR DESCRIPTION
**Pull request by codificat**
## Related Issues and Dependencies

- Triggered by https://github.com/thoth-station/thoth-application/issues/2111

## This introduces a breaking change

- No

## This should yield a new module release

- No

## This Pull Request implements

Use https instead of git for the plugins, and update the versions of the base image and plugins in use.

### CODE REVIEW

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
just one comment to fix pytest.

File: .prow.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ presubmits:
     context: aicoe-ci/prow/pytest
     spec:
       containers:
-        - image: quay.io/goern/thoth-pytest:v0.13.0-dev
+        - image: quay.io/thoth-station/thoth-pytest:v0.14.3
```
[the remark this answers is not in the document]
Comment (codificat) [reply .prow.yaml]: ah, ok, thanks!

**Comment by xtuchyna** [comment]
thx!

/approve

**Review by harshad16 — APPROVED** [review]
lgtm
thanks :100:

**Pull request merged**